### PR TITLE
[WIP] tutor#FollowHelp in favor of 'keywordprg' and K

### PR DESCRIPTION
**Pull request by Pyrohh**
Some links are still broken, hence the WIP.

### CODE REVIEW

File: autoload/tutor.vim [hunk 1]
```diff
@@ -180,22 +179,10 @@ function! tutor#FollowLink(force)
         elseif a:force != 1 && match(l:target, '^@tutor:') > -1
             let l:tutor = matchstr(l:target, '@tutor:\zs.*')
             exe "Tutor ".l:tutor
-        else
-            exe "help ".l:target
```
Comment (fmoralesc) [inline autoload/tutor.vim]: I think we should keep this or create a way so tutorials can link to help files explicitly.

Comment (Pyrohh) [inline autoload/tutor.vim]: maybe.... no rush though, so I'll think about this for a few months.

Comment (fmoralesc) [inline autoload/tutor.vim]: That's the @fmoralesc way of working on PRs. :+1:

Comment (Pyrohh) [inline autoload/tutor.vim]: so you've secretly been working on https://github.com/neovim/neovim/pull/1667 all this time, huh? Slow and steady wins the race I suppose...

Comment (fmoralesc) [inline autoload/tutor.vim]: Indeed!

**Comment by fmoralesc** [comment]
So, @Pyrohh what was still missing from this? I'm feeling like this could be merged already. I plan to improve the tutor syntax this week, make it more similar to markdown's.

**Comment by Pyrohh** [comment]
Some links are "broken", because they depended on the []() syntax, which `K` doesn't recognize. I fix later ok ok.

**Comment by Pyrohh** [comment]
> I plan to improve the tutor syntax this week, make it more similar to markdown's.

:+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1:

**Comment by fmoralesc** [comment]
I merged this into the `devel` branch, so we can have a release branch, the old `master` (I intend to open a PR for neovim to include the version in that branch) and a merging development one (`devel`).

**Comment by Pyrohh** [comment]
Thanks bruv.

> I merged this into the devel branch, so we can have a release branch, the old master (I intend to open a PR for neovim to include the version in that branch) and a merging development one (devel).

So you plan on doing development of tutor-mode in the neovim branch `devel`, which would periodically be merged into master? Not sure.

**Comment by fmoralesc** [comment]
The idea is to develop it here in the `devel` branch, and then periodically
merge it into our `master` branch, which would then be merged into the
neovim runtime. I want to consolidate the master branch here before we
include the recent changes to the tutorials, basically, so I'd prefer to
have this two step mechanism. Do you think it is a bad idea?

**Comment by Pyrohh** [comment]
Ah I see. I think that's a good idea.

**Pull request closed without merging**